### PR TITLE
CORE-260 add terraform module for creating static website

### PR DESCRIPTION
**Pull request by tundeaoni**
https://remerge.atlassian.net/browse/CORE-260

### CODE REVIEW

**Comment by remerge-hal** [comment]
This comment ensures that the correct Slack channel is notified after the team/project label `CORE` has been added to this pull request.

See [this comment](https://remerge.atlassian.net/browse/ITSUP-1973?focusedCommentId=90701) for details.

File: google/static-website/main.tf [hunk 1]
```diff
@@ -0,0 +1,54 @@
+locals {
+  # We have to use dashes instead of dots in the bucket name, because
+  # that bucket is not a website
+  website_domain_name_dashed = replace(var.website_domain_name, ".", "-")
+}
+
+resource "google_compute_backend_bucket" "static" {
+  provider = google-beta
```
Comment (hollow) [inline google/static-website/main.tf:8]: remove beta provider

File: google/static-website/main.tf [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+# Make bucket public by granting allUsers READER access
+resource "google_storage_bucket_iam_member" "default" {
+  bucket = module.website.name
+  role   = "roles/storage.legacyObjectReader"
+  member = "allUsers"
+  # checkov:skip=CKV_GCP_28:Public website
+}
+
+resource "google_dns_record_set" "cname" {
+  depends_on = [module.website]
```
Comment (hollow) [inline google/static-website/main.tf:40]: why does this depend on the module?

**Comment by tundeaoni** [comment]
@hollow addressed your comments

**Review by alexmorten — DISMISSED** [review]
EDIT: Wrong PR

**Review by hollow — APPROVED** [review]

**Pull request merged**